In [2]:
import pandas as pd
import plotly as plt

In [3]:
data = pd.read_csv("./data/psgouv_2023_data.csv")

# Quelques informations sur mon dataset
data.info()

<class 'pandas.DataFrame'>
RangeIndex: 363 entries, 0 to 362
Data columns (total 6 columns):
 #   Column          Non-Null Count  Dtype  
---  ------          --------------  -----  
 0   Région          363 non-null    str    
 1   District        363 non-null    str    
 2   Catégorie       363 non-null    str    
 3   Sous-catégorie  363 non-null    str    
 4   Désignation     363 non-null    str    
 5   Valeur          362 non-null    float64
dtypes: float64(1), str(5)
memory usage: 57.1 KB


##### **Resultats Info**
Mon dataset compte :
- 363 enregistrements
- 06 colonnes dont 05 textuelles et 01 numérique
- Les colonnes sont : 
    * La région, 
    * le District, 
    * la catégorie de l'action
    * la sous-catégorie de l'action
    * la désignation de l'action
    * la valeur


In [4]:
regions = data.Région.unique()
len(regions)

33

In [5]:
districts = data.District.unique()
len(districts)

24

In [6]:
catégories = data.Catégorie.unique()
len(catégories)

3

In [7]:
sous_cats = data["Sous-catégorie"].unique()
len(sous_cats)

6

In [8]:
design = data.Désignation.unique()
len(design)

12

In [9]:
data.describe()

,Valeur
count,362.000000
mean,33231.745856
std,75366.074440
min,0.000000
25%,420.750000
50%,3481.500000
75%,19296.250000
max,738311.000000


#### Mon dataset compte : 
- 33 régions
- 24 districts
- 03 catégories
- 06 sous-catégories
- 12 désignations (Type d'action)
